README.md

```diff
@@ -17,12 +17,17 @@ Not to be confused with [`near-contract-standards`](https://crates.io/crates/nea
 
 ## Build and test
 
+Install `cargo-make` if it is not installed already:
+
+```txt
+cargo install cargo-make
+```
+
+Run tests:
+
 ```
-cargo build --target wasm32-unknown-unknown --release
+cargo test
```

Comment:
@encody  So, `cargo test` rebuilds automatically here but not in the case of Workspaces-rs? 

https://github.com/near/workspaces-rs/pull/176#discussion_r943965007

I guess that's because Workspaces-rs just points to a string like `include_bytes!("../../target/wasm32-unknown-unknown/release/fungible-token.wasm")` at https://github.com/NEARFoundation/near-contract-tools/blob/b7881dedbb1a3c4a18665935b46f8c239095c127/workspaces-tests/tests/fungible-token.rs#L5 and doesn't know how to automatically watch whatever project compiled to that destination Wasm file?

Reply:
Yes, that is correct. `cargo build` is usually pretty clever about what it rebuilds (unless you clean beforehand) so it shouldn't be too expensive to run the make script instead.